### ISSUE TITLE
Feature Request: Auto refresh slideshow with new images

### ISSUE THREAD

**Issue by mcdamion** [chtaylo2/Roku-GooglePhotos#198]
When using the Photo View app on my Roku, if I add a photo within my Google Photos album that is being presented as a slideshow. The Photo View app does not update, I have to back out of the album and restart the slideshow for the changes to show. Is there a fix to this to allow auto refreshing, or is this the way it’s designed. Please advise. Thank you

**Comment by chtaylo2** [comment]
Thats correct. It will only refresh after starting the slideshow again if looking at a normal album. "XXX in History" albums do refresh automatically . I'll add this to the backlog for enhancement consideration down the road.
